notebooks/source/lotka_volterra_multiple.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -q numpyro@git+https://github.com/pyro-ppl/numpyro
</pre><p>CI failed because we need to comment out this command.</p><pre class="ql-syntax" spellcheck="false"># !pip install -q ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>